hack/local-up-cluster.sh

```diff
@@ -62,6 +62,9 @@ CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
 CLOUD_CONFIG=${CLOUD_CONFIG:-""}
 FEATURE_GATES=${FEATURE_GATES:-"AllAlpha=true"}
 
+# enable swagger ui
+ENABLE_SWAGGER_UI=${ENABLE_SWAGGER_UI:-false}
+
```

Comment:
the official site (https://kubernetes.io/kubernetes/third_party/swagger-ui/) seems to be unstable. Could we to make a simple way to use swagger-ui?